ui/pages/swaps/awaiting-swap/awaiting-swap.js

```diff
@@ -283,6 +283,7 @@ export default function AwaitingSwap({
       ) : null}
       <SwapsFooter
         onSubmit={async () => {
+          /* istanbul ignore next */
```

Comment:
total nit: why not `/* istanbul ignore if */`

Reply:
That one would work as well.